components/standings/commons/standings_storage.lua

```diff
@@ -167,6 +168,9 @@ function StandingsStorage.fromTemplateHeader(frame)
 		return
 	end
 
+	data.roundcount = tonumber(data.roundcount) or 1
+	data.finished = Logic.nilOr(Logic.readBoolOrNil(data.finished), true)
```

Comment:
mind this finishes the manual groups on all wikis unless manually overwritten
it also needs
- adjusting in the in the PR summary mentioned templates to pass the param along so that manual input for it actually works
- an information about it to all contributors that if they still use manual group tables on new pages (i.e. where the groups haven't finished yet) they should set `|finished=false` until they are finished

due to having to adjust the slot template on all wikis anyways maybe setting it via those directly would be the better option. This way it could be enabled on a per wiki basis.
Would be `|finished={{{finished|true}}}` in the call of `StandingsData` inside group slot.
(Plus passing along the finished param in `StandingsData`)

Reply:
```suggestion
	data.finished = Logic.readBool(data.finished)
```
plus in templates to pass along plus possible default to true where wanted/needed